HomeTask #05-1 and #05-2

krasnovsasha opened the pull request:
according to the HomeTask 05 changes included uses String.format() function and separating console output by the page

krasnovsasha committed:
docs: project owner information

krasnovsasha committed:
Merge remote-tracking branch 'origin/main'

krasnovsasha committed:
Merge remote-tracking branch 'origin/main'

krasnovsasha committed:
Merge remote-tracking branch 'origin/main'

krasnovsasha committed:
refactor: decorate() method return using String.format()

krasnovsasha committed:
feat: "page"-style console output

hellenaalex commented on src/main/java/com/tcs/edu/decorator/TimestampMessageDecorator.java:
```diff
@@ -18,8 +17,8 @@
+    public static String decorate(String message) {
         messageCount++;
-        return messageCount + " " + Instant.now() + " " + message;
+        return String.format("%s %s %s", messageCount, Instant.now(), message);
```
для messageCount можно использовать %d

hellenaalex replied:
почти полное дублирование кода (за исключением %n---), стоит подумать, как от дублирования можно избавиться

hellenaalex reviewed (changes requested):
Использовать %d для чисел и избавиться от дублирования

krasnovsasha committed:
refactor: code-style improvements
-removed unnecessary duplicated code in decorate() method
-changed %s to %d for int variable messageCount

krasnovsasha committed:
refactor: code-style improvements
-one return against two in decorate() method
